# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: a70e9e75e47c
# Full Hash: a70e9e75e47cb7d6e9753b0d999649967442acac
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-08-25 21:49:19
# Description:
#   Bug 1725743 - Put the use of internal PDF destinations behind a pref, and disable by default due to possible cairo assertions. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123650
# ==============================================================================

diff -r 5e03ffaad62a -r a70e9e75e47c modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Wed Aug 25 16:11:02 2021 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Wed Aug 25 16:28:43 2021 +0000
@@ -10234,6 +10234,14 @@
   value: true
   mirror: always
 
+# Whether we attempt to generate and use document-internal PDF destinations.
+# This currently sometimes results in an internal cairo error, see bug 1725743;
+# disabled by default until that is resolved.
+- name: print.save_as_pdf.internal_destinations.enabled
+  type: RelaxedAtomicBool
+  value: false
+  mirror: always
+
 # The default DPI for printing.
 #
 # For PDF-based output, DPI should ideally be irrelevant, but in fact it is not
